### ISSUE TITLE
Qgis doesn't launch in debian bookworm

### ISSUE THREAD

**Issue by desputinski** [qgis/QGIS#54946]
### What is the bug or the crash?

Hi, I have qgis version 3.32 Lima installed in debian. but it doesn't execute/launch. I don't get a satisfying error message.

So I tried to install the flatpak-version but that also won't run. Error message:

`flatpak run org.qgis.qgis
ERROR: ld.so: object 'libgtk3-nocsd.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file)
: ignored.
Warning: Qt: Session management error: Could not open network socket`

Can anyone help me with that? I tried emptying cache and /tmp and renaming the user dictionary. Didn't work.

### Steps to reproduce the issue

type qgis in the terminal, press enter

or:

flatpak run org.qgis.qgis

or:

qgis --noplugins

### Versions

3.32 with debian stable (bookworm)

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

**Comment by desputinski** [comment]
Hm, Can anyone reproduce the error? I asked in my (German) debian forum : https://debianforum.de/forum/viewtopic.php?p=1341848#p1341848

And the other user, running debian stable was having the same issues with qgis not starting.

**Comment by desputinski** [comment]
Help! I need to work with qgis :(

**Comment by desputinski** [comment]
Ah, it turns out, that Qgis is not visible if I'm running Compiz/Emerald. If Xfwm4 is active, it works!

**Comment by desputinski** [comment]
And it may be xfce related. So I'll try there too:
https://gitlab.xfce.org/xfce/xfdesktop/-/issues/271

**Comment by desputinski** [comment]
Ok, so I testet how Qgis runs under MATE instead of XFCE and there everything looks fine.

**Comment by alexbruy** [comment]
Flatpack is not officially supported by QGIS team. Does QGIS installed from the debian repository works for you? Probably this is a flatpack issue.

**Comment by desputinski** [comment]
> Flatpack is not officially supported by QGIS team. Does QGIS installed from the debian repository works for you? Probably this is a flatpack issue.

No, the installed version had the same problem. I think it is a QT issue under XFCE. But now I don't have the problem anymore because I'm using MATE. If you want to test the problem, install qgis under Debian Stable/Bookworm with XFCE.
